README.md

```diff
@@ -1,9 +1,9 @@
+<<<<<<< HEAD
+Somebody please make this readme beautiful
+
+true
+=======
```

Comment:
Everything above this line seems to be artifacts from when you were merging. i would just delete everything above #Team Space Platypi.